src/ingest_validation_tools/table-schemas/includes/fields/sc_isolation_quality_metric.yaml

```diff
@@ -0,0 +1,15 @@
+# scrnaseq-hca-v0.yaml
+- name: sc_isolation_quality_metric
+  constraints:
+    required: false
```

Comment:
Since sc_isolation_quality_metric is not required (constraint: FALSE) for HCA but TRUE for other data sources, shouldn't both of these sections to be left as is?

Reply:
No: The `required: false` will override the base definition.